packages/app-mobile/metro.config.js

```diff
@@ -22,6 +22,16 @@ const localPackages = {
 	'@joplin/react-native-alarm-notification': path.resolve(__dirname, '../react-native-alarm-notification/'),
 };
 
+const remappedPackages = {
+	...localPackages,
+};
+
+// Remap packages like
```

Comment:
Could you add a comment to explain why this is needed please?

Reply:
This has been done!